src/iop/colormapping.c

```diff
@@ -268,7 +267,7 @@ static void get_cluster_mapping(const int n, float mi[n][2], const float wi[n],
 
 
 // inverse distant weighting according to D. Shepard's method; with power parameter 2.0
-static void get_clusters(const float *col, const int n, float mean[n][2], float *weight)
```

Comment:
Arrays in function arguments decay to pointers. This rule applies to arrays with size known at compile time as well as to variable length arrays. Thus, the above is equivalent to

    static void get_clusters(const float *col, const int n, float *mean[2], float *weight) {...}